src/vendor/wordpress/styles/_core-blocks.scss

```diff
@@ -281,3 +281,16 @@ $wp-button-gap: size.$spacing-gap-button-group-default;
     text-align: left;
   }
 }
+
+/**
+ * Gutenberg block: Separator
+ *
+ * The default separator is intended to be partial-width, so we apply those
+ * styles selectively here rather than un-applying them to the other modifiers.
+ */
+
+.wp-block-separator.is-style-default {
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
Is our browser support good enough to use `margin-inline`?

Reply:
@spaceninja Good question. It looks like we currently have 104 occurrences of traditional margin properties in the project already, so I think I'll create a separate issue about investigating this. Maybe also look into a linting error depending on where that lands?

Reply:
@spaceninja Bam: #1576 1576